CleanWeather/Database/CityDatabase.swift

```diff
@@ -0,0 +1,58 @@
+//
```

Comment:
Zróbmy to inaczej, bo zapis do pliku będzie trudniejszy do testowania. Uzyj userdefaultsow i konwersji do JSONA:
https://www.hackingwithswift.com/example-code/system/how-to-load-and-save-a-struct-in-userdefaults-using-codable

Reply:
i nazwij to FavouriteCityRepository

w inicie przyjmuj obiekt UserDefaults, który creator wezmie z Applowego singletona: USerDefaults.shared